src/components/UserDashboard/index.js

```diff
@@ -60,17 +62,10 @@ export default compose(
         <H1 mb={'35px' /* On removing Beta Message change to 44px */}>
           Welcome, {loggedInUser.firstName}!
         </H1>
-        <Column>
-          <div
-            css={`
-              display: flex;
-              overflow: hidden;
-            `}
-          >
-            <MySavedQueries {...{ api, loggedInUser, theme, profileColors }} />
-            <Notifications />
```

Comment:
not sure what this is but what happened to this guy? :P

Reply:
That's just a block with a h3 of 'notifications' and an opacity of 0.
I haven't seen it in any designs